src/powfacpy/base_interface.py

```diff
@@ -1003,8 +1003,27 @@ def get_name_with_ending(objects):
     if not type(objects) == list:
       objects = [objects]
     return [x.GetFullName().split('\\')[-1] for x in objects]
-    
 
-if __name__ == "__main__":
-  print("ok")
-  
\ No newline at end of file
+
+def set_attr_of_obj(obj, attributes:dict):
+  """Set attributes of object.
+  The difference to set_attr of PFBaseInterface is that
+  this method only accepts PF objects (and not path strings)
+  and is slightly more performant.
+  """
+  for attr,value in attributes.items():
+    obj.SetAttribute(attr,value)
+
+def set_attr_of_objects(objects: Iterable,attributes: Iterable):
```

Comment:
better: set_attributes_of_objects

Reply:
Here objects is used in the method name to make it obvious that these are multiple objects. I prefer that to the abbreviation objs or obs.